testing/mu/APKBUILD

```diff
@@ -1,40 +1,41 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Contributor: stef <l0ls0fo2i@ctrlc.hu>
+# Contributor: Eivind Uggedal <eu@eju.no>
 # Maintainer:
```

Comment:
If you use regularly would you consider becoming the maintainer?

Reply:
I'm not using it, and I won't maintain it nor modernize and do the fixups. I just added the mu4e subpackage for @telmich (maybe he wants to maintain it?).